legacy/application/models/StreamSetting.php

```diff
@@ -283,7 +283,7 @@ public static function getDefaults($prefix)
         $config = Config::getConfig();
 
         return [
-            'host' => $config['baseUrl'],
+            'host' => 'localhost',
```

Comment:
Is this used to generate the listening url on the mini site or from the UI?

Reply:
Hmm, I guess I'm going a bit too fast.

I'll have a look, but I think you are right.

Reply:
Reverting back, if we change this I'll make a dedicated PR.